### ISSUE TITLE
Wrong link reference definition for advanced markdig

### ISSUE THREAD

**Issue by qinezh** [lunet-io/markdig#159]
For the markdown such like:

```
![scenario image][scenario]
## Scenario
[scenario]: ./scenario.png
```

The output from markdig and advanced markdig are different: https://babelmark.github.io/?text=!%5Bscenario+image%5D%5Bscenario%5D%0A%23%23+Scenario%0A%5Bscenario%5D%3A+.%2Fscenario.png

**Comment by xoofx** [comment]
Thanks, this should be fixed by commit cd17a54  as part of `0.14.4+`

**Comment by xoofx** [comment]
Actually, didn't check carefully as it is introducing a regression. Will have to dig into this more.

**Comment by xoofx** [comment]
Test comment with github:

![scenario image][scenario]
## Scenario
[scenario]: ./scenario.png

**Comment by qinezh** [comment]
Thanks for the quick response, if there's anything need from my side please let me know : )

**Comment by xoofx** [comment]
This is fixed by commit 8d4394e
